adapters/bmtm/params_test.go

```diff
@@ -0,0 +1,44 @@
+package bmtm
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
```

Comment:
Please consider changing this to using `assert.NoError` as well. Similar to above